test_unstructured_ingest/metrics/metrics-json-manifest.txt

```diff
@@ -1,3 +1,4 @@
+handbook-1p.docx.json
 example-10k.html.json
 IRS-form-1987.pdf.json
 science-exploration-1p.pptx.json
```

Comment:
ohh, curious, why do we list all of these in the manifest, but only actually expect example-10k and the IRS form? also -- why adding handbook-1p if it's not in the all-docs-cct.tsv?

Reply:
We are now combining all manifest for both cct and element type. I am working on creating gold standard for element type and should be in soon on another pr!